Makefile

```diff
@@ -5,7 +5,7 @@ CURL ?= $(shell command -v curl 2> /dev/null)
 MM_DEBUG ?=
 MANIFEST_FILE ?= plugin.json
 GOPATH ?= $(shell go env GOPATH)
-GO_TEST_FLAGS ?= -race
```

Comment:
Why are you removing the `-race` here? we want to check the race conditions in CI

Reply:
Hey @jespino, this was an accidental change.